client/landing/gutenboarding/components/signup-form/index.tsx

```diff
@@ -199,90 +199,24 @@ const SignupForm = ( { onRequestClose }: Props ) => {
 
-	return (
-		<Modal
-			className={ 'signup-form' }
-			title={
-				isAnchorFmSignup
-					? __( 'Create your podcast site with WordPress.com' )
-					: __( 'Save your progress' )
-			}
-			onRequestClose={ closeModal }
-			focusOnMount={ false }
-			isDismissible={ false }
-			overlayClassName={ 'signup-form__overlay' }
-			// set to false so that 1password's autofill doesn't automatically close the modal
-			shouldCloseOnClickOutside={ false }
-		>
-			<SignupFormHeader onRequestClose={ closeModal } />
-
-			<div className="signup-form__body">
-				<h1 className="signup-form__title">
-					{ isAnchorFmSignup
-						? __( 'Create your podcast site with WordPress.com' )
-						: __( 'Save your progress' ) }
-				</h1>
-
-				<form onSubmit={ handleSignUp }>
-					<fieldset className="signup-form__fieldset">
-						<legend className="signup-form__legend">
-							<p>
-								{ isAnchorFmSignup
-									? __( 'Create a WordPress.com account and start creating your free site.' )
-									: __( 'Enter an email and password to save your progress and continue.' ) }
-							</p>
-						</legend>
-
-						<TextControl
-							value={ emailVal }
-							disabled={ isFetchingNewUser }
-							type="email"
-							onChange={ setEmailVal }
-							placeholder={ __( 'Email address' ) }
-							required
-							autoFocus={ ! isMobile } // eslint-disable-line jsx-a11y/no-autofocus
-						/>
-
-						<TextControl
-							value={ passwordVal }
-							disabled={ isFetchingNewUser }
-							type="password"
-							autoComplete="new-password"
-							onChange={ setPasswordVal }
-							placeholder={ __( 'Password' ) }
-							required
-						/>
-
-						{ errorMessage && (
-							<Notice className="signup-form__error-notice" status="error" isDismissible={ false }>
-								{ errorMessage }
-							</Notice>
-						) }
-
-						<div className="signup-form__footer">
-							<p className="signup-form__login-link">
-								<span>{ __( 'Already have an account?' ) }</span>{ ' ' }
-								<Button className="signup-form__link" isLink href={ loginUrl }>
-									{ __( 'Log in' ) }
-								</Button>
-							</p>
-
-							<p className="signup-form__link signup-form__terms-of-service-link">{ tos }</p>
-
-							<ModalSubmitButton disabled={ isFetchingNewUser } isBusy={ isFetchingNewUser }>
-								{ __( 'Create account' ) }
-							</ModalSubmitButton>
-
-							<p className="signup-form__link signup-form__terms-of-service-link signup-form__recaptcha_tos">
-								{ recaptcha_tos }
-							</p>
-						</div>
-					</fieldset>
-				</form>
-			</div>
-
-			<div id="g-recaptcha"></div>
-		</Modal>
+	const displayProps = {
+		closeModal,
+		emailVal,
+		errorMessage,
+		handleSignUp,
+		isFetchingNewUser,
+		isMobile,
+		loginUrl,
+		passwordVal,
+		recaptcha_tos,
+		setEmailVal,
+		setPasswordVal,
+		tos,
+	};
+	return isAnchorFmSignup ? (
+		<SignupDisplayAnchor { ...displayProps } />
```

Comment:
Could these maybe be `SignupDefaultLayout` and `SignupAnchorLayout`? That might be slightly more clear on first glance.

Reply:
Meh.